site/app/templates/banner/Banner.twig

```diff
@@ -0,0 +1,37 @@
+<div class="content">
+
+<button class="Upload_Banner btn btn-primary" onclick="newUploadBanner()"> Upload an Event Banner </button>
+
+<br>
+<br>
+
+<input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
+{% for bannerImage in banners %}
+    {% set imageName = bannerImage.getName() %}
+    {% set imagePath = bannerImage.getPath() %}
+    {% set id = bannerImage.getId() %}
+    {% set description = bannerImage.getExtraInfo() %}
+    {% set releaseDate = bannerImage.getReleaseDate().format('Y-m-d') %}
+    {% set closeDate = bannerImage.getClosingDate().format('Y-m-d') %}
+
+    {# Do something with the attributes... #}
+    <div>
+		<div class="image-info" style="display: inline-flex; align-items: center;">
+		  <h3 style="margin-right: 10px;">{{ imageName }}</h3>
+		  <a onclick='deleteBannerImage("{{ csrf_token }}", "{{ imageName }}", "{{ imagePath }}", "{{ description }}", "{{ releaseDate }}", "{{ closeDate }}");'>
```

Comment:
Might be good to add the twig escape filter for js. We do this in a couple places around the codebase. Search for `|e('js')` and you will see some examples.